problems283_c.py
